iroha/src/config.rs

```diff
@@ -158,7 +158,7 @@ impl Configuration {
 
     /// Set `public_key` and `private_key` configuration parameters - will overwrite the existing one.
     pub fn key_pair(&self) -> (PublicKey, PrivateKey) {
-        (self.public_key.clone(), self.private_key)
+        (self.public_key.clone(), self.private_key.clone())
     }
```

Comment:
The doc comment should be corrected as the method no longer sets the `key_pair`, but gets it instead

Reply:
Also why not return a `KeyPair` type instead of tuple?

Reply:
Just tried to minimize changes in our code for now. Will fix doc